docs/manual/physicslist.md

```diff
@@ -1,46 +1,153 @@
 # Physics list
 
-:::{todo}
+_remage_ uses a custom modular physics list built on top of Geant4’s standard
+physics constructors. It mostly is based on the physics list implemented in
+_MaGe_. It combines:
+
+- Configurable **electromagnetic physics** (standard, low-energy, Livermore,
+  Penelope, polarized)
+- Optional **optical physics** (scintillation, Cherenkov, absorption, Rayleigh,
+  WLS)
+- Configurable **hadronic physics lists**, including high-precision neutron
```

Comment:
I don't think "high-precision" is the right adjective here

Reply:
okay, I checked. NeutronHP really means high-precision neutrons